src/Microsoft.OData.Core/UriParser/SemanticAst/ExpandedCountSelectItem.cs

```diff
@@ -0,0 +1,53 @@
+﻿//---------------------------------------------------------------------
+// <copyright file="ExpandedCountSelectItem.cs" company="Microsoft">
+//      Copyright (C) Microsoft Corporation. All rights reserved. See License.txt in the project root for license information.
+// </copyright>
+//---------------------------------------------------------------------
+
+namespace Microsoft.OData.UriParser
+{
+    using Aggregation;
+    using Microsoft.OData.Edm;
+
+    /// <summary>
+    /// This represents one level of expansion for a particular expansion tree.
+    /// </summary>
+    public sealed class ExpandedCountSelectItem : ExpandedReferenceSelectItem

```

Comment:
Do you think it's better to derive from ExpandedReferenceSelectItem?

Or shall we have a base NavigationSelectItem, and others derived from it?

Reply:
The semantics are the same though.
Also, `ExpandedNavigationSelectItem` inherits from the `ExpandedReferenceSelectItem`